azure-pipelines.yml

```diff
@@ -182,27 +182,42 @@ stages:
               arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
             env:
               Token: $(dn-bot-dnceng-artifact-feeds-rw)
-        - script: eng\common\cibuild.cmd
+        - script: eng\CIBuild.cmd
+            -restore
+            -build
+            -sign
+            -pack
+            -publish
             -configuration $(_BuildConfig)
             -msbuildEngine dotnet
             -prepareMachine
-            -integrationTest
             $(_BuildArgs)
             $(_PublishArgs)
             $(_InternalRuntimeDownloadArgs)
           name: Build
           displayName: Build
           condition: succeeded()
-        - script: eng\common\cibuild.cmd
+        - script: eng\cibuild.cmd
```

Comment:
The file itself is `CIBuild.cmd` (pascal cased), will that cause issues in Linux/Mac builds or is this part not run there?

Reply:
This only runs on the Windows build. Also, `eng\common\CIBuild.cmd` is the name of the existing script, not the all lower-case that was previously there, so we already know it doesn't matter.